### DISCUSSION TITLE
SEO friendly route change without full-page refresh

### DISCUSSION THREAD

**Discussion by abolfazl-zargar** [quasarframework/quasar#17288] [CLI - SSR mode] [answered]
because of SEO i should use a href
but it refreshes full-page and is not user friendly (I want animated transitions instead of page refresh)
nuxt has nuxt-link for this issue
what is the alternative of nuxt-link in quasar

**Comment by stefanvanherwijnen** [comment, marked as answer]
router-link? ([vue-router](https://router.vuejs.org/))
